benchopt/tests/test_benchmark_features.py

```diff
@@ -99,3 +100,32 @@ def raise_error():
             )
     finally:
         os.environ['BENCHOPT_RAISE_INSTALL_ERROR'] = prev_value
+
+
+def test_ignore_hidden_files():
```

Comment:
I just simplified a bit this test but it looks super good! thanks @antoinecollas